src/stories/migration.mdx

```diff
@@ -26,6 +26,22 @@ Breaking Changes
   - Radio Button Group , Number Input, Text Input , Text Area
     - Added a dedicated slot for tooltip.
     - `label` slot has been updated to a `label` string prop
+- Updated date, time, and date range picker components
+  - Utilizes Flatpickr third-party date picker library to standardize cross-browser date, time, and date range pickers, and to support theming and locale
+  - The locale can now be set via the `locale` attribute
```

Comment:
Should this be "can" or "must"?

Reply:
✅ -- totally reorganized this migration guide to be more of a checklist for devs